### PR TITLE
pass `options` to `it` within `run_spec`

### PR DESCRIPTION
**Pull request by a-lavis**
## Problem
It would be nice to be able to pass options to `it` through `run_test!`, such as `focus: true` like so:
```ruby
run_test! focus: true
```
This dovetails nicely with the work that is already being done to add strict schema validation.

## Solution
Pass `**options` to `it`, instead of merging it with `metadata`.

### CODE REVIEW

File: rswag-specs/lib/rswag/specs/example_group_helpers.rb [hunk 1]
```diff
@@ -122,23 +122,25 @@ def example(mime, name, value, summary=nil, description=nil)
       # @param &block [Proc] you can make additional assertions within that block
       # @return [void]
       def run_test!(**options, &block)
+        options[:rswag] = true unless options.key?(:rswag)
```
Comment (IlkhamGaysin) [inline rswag-specs/lib/rswag/specs/example_group_helpers.rb:125]: let's also add an explanation section to README that it's possible to pass options that are not related to rswafg itself?

Comment (a-lavis) [reply rswag-specs/lib/rswag/specs/example_group_helpers.rb:125]: Just committed: ed34ecef0fc48fe1c905f51abf1aa2780b482f41
Let me know your thoughts 😃

File: README.md [hunk 2]
```diff
@@ -189,6 +189,14 @@ If you've used [Swagger](http://swagger.io/specification) before, then the synta
 
 Take special note of the __run_test!__ method that's called within each response block. This tells rswag to create and execute a corresponding example. It builds and submits a request based on parameter descriptions and corresponding values that have been provided using the rspec "let" syntax. For example, the "post" description in the example above specifies a "body" parameter called "blog". It also lists 2 different responses. For the success case (i.e. the 201 response), notice how "let" is used to set the blog parameter to a value that matches the provided schema. For the failure case (i.e. the 422 response), notice how it's set to a value that does not match the provided schema. When the test is executed, rswag also validates the actual response code and, where applicable, the response body against the provided [JSON Schema](http://json-schema.org/documentation.html).
 
+If you want to add metadata to the example, you can pass keyword arguments to the __run_test!__ method:
+
+```ruby
+response '201', 'blog created' do
+  run_test! focus: true
+end
```
Comment (IlkhamGaysin) [inline README.md:198]:
```suggestion
# to run particular test case
response '201', 'blog created' do
  run_test! focus: true
end

# to write vcr cassette
response '201', 'blog created' do
  run_test! vcr: true
end
```

Comment (IlkhamGaysin) [reply README.md:198]: I've added one more real word example
